package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hyperproof/hypersync-models",
-  "version": "4.1.0",
+  "version": "4.1.1",
```

Comment:
Do we need to have a corresponding update to the [hypersync-sdk](https://github.com/Hyperproof/hypersync-sdk/) package as well?

Reply:
No.  The hypersync-sdk is not yet using hypersync-models.  That's going to happen when we bump it to 2.0.